### PR TITLE
Using local TS node module

### PR DESCRIPTION
**Pull request by bdrazhzhov**
I've added ability to use TS node module (if it's available) with fallback to internal module.
Now 1.4.x is stable branch and it does not include typescript.js file and I decided to use tsc.js as source file.

### CODE REVIEW

**Comment by Nekromancer** [comment]
Awsome! It's pretty sad that this repository is kinda abandoned. Solution you've prepared is great

**Comment by tkawachi** [comment]
@bdrazhzhov Could you help maintaining this repository? I'm willing to add you as a contributor.

**Comment by Nekromancer** [comment]
@tkawachi He has github account only for this fork, so i'm not sure if he's active. Could you accept this pull reqest? It would solve a lot of problems in https://github.com/typescript-ruby/typescript-rails repo

File: lib/typescript-src.rb [hunk 1]
```diff
@@ -7,7 +7,15 @@ module Src
     class << self
       # @return [Pathname]
       def typescript_path
-        @typescript_path ||= ::Pathname.new(File.dirname(__FILE__)).join('typescript-src/support/typescript')
+        unless @typescript_path
+          begin
+            @typescript_path = ::Pathname.new(`npm root -g`.strip! + '/typescript')
```
Comment (tkawachi) [inline lib/typescript-src.rb]: This doesn't throw an exception even if the directory doesn't exist.

**Comment by bdrazhzhov** [comment]
@tkawachi I'm new in RoR and I would like to help with maintaining this repository.

**Comment by tkawachi** [comment]
Thank you @bdrazhzhov. I've invited you to this organisation. I hope you commit/merge pull requests. Currently none of organisation owners are active.

BTW, I think it's not good to always detect npm installed tsc. It's better to be able to configure by library users.

**Comment by bdrazhzhov** [comment]
Oh... Thank you! Will implement it.

**Comment by bdrazhzhov** [comment]
@tkawachi, could you provide me permission to push the gem to rubygems.org? What do you need to know from me for permission providing?

**Comment by tkawachi** [comment]
@bdrazhzhov According to [the FAQ](http://help.rubygems.org/discussions/problems/72-how-can-i-let-multiple-people-push-out-a-new-version-of-a-gem), your email address which is registered to rubygems.org is needed.

**Comment by bdrazhzhov** [comment]
I used templar8@gmail.com for rubygems.org.
I would like to publish typescript-node-ruby and typescript-rails gems too. Could you provide me permissions for these gem publishing to rubygems.org?

**Comment by tkawachi** [comment]
I've added you to [typescript-src](https://rubygems.org/gems/typescript-src) and [typescript-node](https://rubygems.org/gems/typescript-node) as an owner.
I'm not an owner of [typescript-rails](https://rubygems.org/gems/typescript-rails), could you add @bdrazhzhov as an owner, @gfx @klaustopher ?

**Comment by gfx** [comment]
@bdrazhzhov Added you to owners of typescript-rails. Thanks to maintain it.

**Pull request merged**